[2025-04-14T15:00:15.682+0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-14T15:00:15.692+0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_world.hello_world_task scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-04-14T15:00:15.700+0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_world.hello_world_task scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-04-14T15:00:15.701+0300] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-14T15:00:15.722+0300] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): hello_world_task> on 2025-04-13 00:00:00+00:00
[2025-04-14T15:00:15.742+0300] {standard_task_runner.py:72} INFO - Started process 42901 to run task
[2025-04-14T15:00:15.792+0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'hello_world', 'hello_world_task', 'scheduled__2025-04-13T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/sample_hello_world.py', '--cfg-path', '/tmp/tmp2a9ywqoa']
[2025-04-14T15:00:15.802+0300] {standard_task_runner.py:105} INFO - Job 18: Subtask hello_world_task
[2025-04-14T15:00:16.139+0300] {task_command.py:467} INFO - Running <TaskInstance: hello_world.hello_world_task scheduled__2025-04-13T00:00:00+00:00 [running]> on host DESKTOP-1SBO1JU.
[2025-04-14T15:00:16.269+0300] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='tasneemahmed' AIRFLOW_CTX_DAG_ID='hello_world' AIRFLOW_CTX_TASK_ID='hello_world_task' AIRFLOW_CTX_EXECUTION_DATE='2025-04-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-13T00:00:00+00:00'
[2025-04-14T15:00:16.270+0300] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-14T15:00:16.302+0300] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-14T15:00:16.304+0300] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello world once again!']
[2025-04-14T15:00:16.312+0300] {subprocess.py:99} INFO - Output:
[2025-04-14T15:00:16.326+0300] {subprocess.py:106} INFO - Hello world once again!
[2025-04-14T15:00:16.327+0300] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-14T15:00:16.356+0300] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-14T15:00:16.357+0300] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=hello_world, task_id=hello_world_task, run_id=scheduled__2025-04-13T00:00:00+00:00, execution_date=20250413T000000, start_date=20250414T120015, end_date=20250414T120016
[2025-04-14T15:00:16.402+0300] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-14T15:00:16.411+0300] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-14T15:00:16.412+0300] {local_task_job_runner.py:245} INFO - ::endgroup::
